### PR TITLE
Format bug/xsup 22731

### PR DESCRIPTION
**Pull request by eyalpalo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-22731)

### CODE REVIEW

File: Packs/CommonScripts/Scripts/FormatURL/FormatURL.py [hunk 1]
```diff
@@ -707,7 +707,9 @@ def correct_and_refang_url(url: str) -> str:
         schemas = re.compile("(meow|hxxp)", re.IGNORECASE)
         url = url.replace("[.]", ".")
         url = url.replace("[:]", ":")
-        url = re.sub(schemas, "http", url)
+        lower_url = url.lower()
```
Comment (Ni-Knight) [inline Packs/CommonScripts/Scripts/FormatURL/FormatURL.py:710]: no, no lowering the URL.

**Review by Ni-Knight — APPROVED** [review]

File: Packs/CommonScripts/ReleaseNotes/1_11_58.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+#### Scripts
+
+##### ExtractDomainAndFQDNFromUrlAndEmail
+
+- Fixed an issue where url with **hxxp** or **mewo** were replaced with **http** in cases they shouldn't.
```
Comment (Ni-Knight) [inline Packs/CommonScripts/ReleaseNotes/1_11_58.md:6]:
```suggestion
- Fixed an issue where url with **hxxp** or **meow** were replaced with **http** in cases they shouldn't.
```
typo

File: Packs/CommonScripts/ReleaseNotes/1_11_58.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+##### FormatURL
+
+- Fixed an issue where url with **hxxp** or **mewo** were replaced with **http** in cases they shouldn't.
```
Comment (Ni-Knight) [inline Packs/CommonScripts/ReleaseNotes/1_11_58.md:9]:
```suggestion
- Fixed an issue where url with **hxxp** or **meow** were replaced with **http** in cases they shouldn't.
```

**Review by dantavori — APPROVED** [review]

**Comment by eyalpalo** [comment]
@Ni-Knight FYI

**Pull request merged**